src/resources/elements/popupNotifications/popupNotifications.ts

```diff
@@ -65,6 +66,7 @@ export class PopupNotifications {
     this.resolveToClose = resolve;
     this.showing = true;
     this.countdownRunning = true;
+    this.title = config.title;
```

Comment:
What is `title` for, where is used?  There should be an example in the storybook.

Reply:
There was no way to change what the title of the popup notification and the requirements said there had to be a different title. I can add an example to the storybook.